blog/20250108-introducing-geojupyter/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introducing GeoJupyter"
+date: "2025-01-08"
+description: |
+  We believe that geospatial data analysis and exploration should be free, open source,
+  and accessible for everyone!
+author:
+  - name: "Matt Fisher"
+    orcid: "0000-0003-3260-5445"
+categories:
+  - "Announcements"
+---
+
+## :jigsaw: Need
+
+**We believe that geospatial data analysis and exploration should be free, open source,
+and accessible for everyone!**
+
+Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and
```

Comment:
I would add some language that acknowledges this is a first approximation/simplification, b/c we know that all modern desktop GIS GUIs offer scripting options, and that many code-centric tools also integrate with some GUI elements (e.g. Qiusheng's [WhiteBox](https://github.com/opengeos/whiteboxgui) and I'm sure others).

But we're simply framing the basics of the picture, so we'll go with this simplified version, we're not trying to get into those finer points - our goal is to leverage the architecture of Jupyter to support better modern worfklows.

Reply:
:100: good call! Thinking more about this.

Reply:
873ce1f